<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmsDialog</class>
 <widget class="QDialog" name="FirmsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справочник фирм</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBoxNetwork">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Сеть АЗС:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="toolButtonAdd">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Добавить...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resourse.qrc">
         <normaloff>:/Images/new_document_icon.png</normaloff>:/Images/new_document_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QToolButton" name="toolButtonExit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Закрыть&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resourse.qrc">
         <normaloff>:/Images/exit_stock_icon.png</normaloff>:/Images/exit_stock_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="toolButtonEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Изменить...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resourse.qrc">
         <normaloff>:/Images/edit_icon.png</normaloff>:/Images/edit_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="4" colspan="3">
      <widget class="QTableView" name="tableView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="checkBoxIsActive">
       <property name="text">
        <string> Только и работе</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
